ansible/roles/logstash-openshift/README.md

```diff
@@ -0,0 +1,17 @@
+# Role to provision logstash
+
+The role isn't idempotent at the moment.
+
+## Provisioning
+
+Run the role to provision `logstash`.
+
+## Deletion
+
+Run the role, pass the variable `logstash_deploy=false` to the role to get
+logstash wiped completely.
+
+## Add your configuration to logstash
+
+Add your configuration files to [{{ role_path }}/files/conf.d](files/conf.d).
```

Comment:
Do we add configuration files before we deploy?  If we have this already deployed, do we redeploy to add new configuration files?

Logstash has the ability to reload its configuration without restarting.  Are we taking advantage of that?